TijanaSonjic/assignment2/Q1.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+#include <iostream>
+using namespace std;
+
+class Node{
+private:
+	Node* leftChild;
+	Node* rightChild;
+	int info;
+public:
+	Node(int info, Node* leftChild = nullptr, Node* rightChild = nullptr) {
+		this->info = info;
+		this->leftChild = leftChild;
+		this->rightChild = rightChild;
+	}
+	
+	void distroy() {
+		if (leftChild) {
+			leftChild->distroy();
+			leftChild = nullptr;
+		}
+		if (rightChild) {
+			rightChild->distroy();
+			rightChild = nullptr;
+		}		
+	}
+
+	~Node() {
+		distroy();
+	}
+	
+	Node* insertLeftChild(Node* node) {
+		this->leftChild = node;
+
+		return this;
+	}
+
+	Node* insertRightChild(Node* node) {
+		this->rightChild = node;
+
+		return this;
+	}
+
+	int getInfo() {
+		return info;
+	}
+
+	Node* getLeftChild() {
+		return leftChild;
+	}
+
+	Node* getRightChild() {
+		return rightChild;
+	}
+};
+
+// Function will return false if key is not in the tree.
```

Comment:
Tip:
Avoid double negative (in both code and comments):
`// Function will return true if key is in the tree.`
Is easier to understand.

Reply:
I will do it in the future, thank you for drawing attention to that.